### DISCUSSION TITLE
Color of route sections and curvature

### DISCUSSION THREAD

**Discussion by SF4User** [devemux86/cruiser#158] [Q&A] [answered]
I'm "worked" with Kurviger K2 for the last two years, i.e. with Cruiser from devemux86 (de facto).
Now they both go their separate ways . . .
What I now miss in the "Original" Cruiser is this function:
![Kurvigkeiten](https://user-images.githubusercontent.com/132825580/236874092-8044b33e-661a-4b69-acb1-babf06d3af7c.png)
Will it be possible again, to freely set the color of route sections and curvyness? in one of next updates?

**Comment by devemux86** [comment]
@SF4User thanks for the idea!

This is possible, I will put it in my list.

**Reply by 1maxi2** [reply]
Das würde ich auch sehr begrüßen !
Mir geht es gar nicht um die verschiedenen Farben. 
Aber die verschiedenen Routingprofile
vermisse ich sehr da ich früher auch kurviger 2 verwendet habe

**Comment by 0709wiwiwi** [comment]
So that the menu choices remain clear only active when importing .kurviger files ? Possible ?

**Reply by 1maxi2** [reply]
Habe auch noch K2 zum planen da diese Vermeidungen und Routenoptionen zur Auswahl hat. Navigation dann mit Cruiser was übrigens hervorragend funktioniert 😊
Die entscheidende Frage ist wie lange wird K2 noch funktionieren ?

**Reply by WolfgangZ42** [reply]
> Die entscheidende Frage ist wie lange wird K2 noch funktionieren ?

Kannst Du leicht feststellen:
<img width="600" src="https://github.com/devemux86/cruiser/assets/121806238/ed9739de-8eab-48ab-a508-668d1f5d0211"/>

**Reply by 1maxi2** [reply]
Ich hab kein Abo. Hab die App damals mit Einmal Zahlung gekauft.
Sollte Cruiser die Routenoptionen und Vermeidungen wie in kurviger alt übernehmen dann benötige ich die alte kurviger app nicht mehr. Aber diese Funktion war so genial das ich sie unbedingt haben will.
Ansonsten funktioniert die Cruiser app nahezu perfekt.Letzte Ausfahrt ca. 300 km war wieder ein Beispiel dafür. Sinnvolle Features wie die Warnung bei niedrigen Akkustand machen die App so einzigartig 😄

**Reply by WolfgangZ42** [reply]
> Ich hab kein Abo

Dann hast Du auch nicht K2 sondern K1 (Kurviger Pro). K1 wird noch solange funktionieren bis Android xx oder der Kurviger Server dem einen Riegel vorschiebt.

**Reply by 1maxi2** [reply]
O.k. Sorry habe tatsächlich K1

**Comment by devemux86** [comment]
Kurviger routing can not be used in Cruiser, please see:
Kurviger-Routing kann nicht in Cruiser verwendet werden, siehe:

- https://github.com/devemux86/cruiser/discussions/102

**You can import Kurviger routes** and the navigation rerouting does not change the Kurviger route.
**Sie können Kurviger-Routen importieren** und die Umleitung der Navigation ändert die Kurviger-Route nicht.

**Comment by devemux86** [comment]
Different colors per route profile will be available for BRouter, Kurviger and any router with route profiles:
Verschiedene Farben pro Routenprofil werden für BRouter, Kurviger und jeden Router mit Routenprofilen verfügbar sein:

<img src="https://github.com/devemux86/cruiser/assets/3484020/b0729c25-c12f-4d46-8b95-6b5012e6164f" width="33%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/c3727418-1b8c-4d68-ba43-223d845c0b28" width="33%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/46df1358-2b95-4169-b1d5-ca4c5fb66d7b" width="33%"/>

**Reply by SF4User** [reply]
I have the Cruiser App subscription (brand new) . . . but the top tab [Colors] does not appear in my menu. App is fresh updated.
Fail in app? or is this still a preview?

**Reply by devemux86** [reply]
> I have the Cruiser App subscription (brand new)

@SF4User thanks for your support!

> the top tab [Colors] does not appear in my menu

The above images are a preview, it will be available in the next Cruiser update.

**Comment by devemux86** [comment]
Note that navigation uses one color: the "Route color".

**Comment by 0709wiwiwi** [comment]
Colour of route sections and curvature
With kurviger files, there is a defined limited range. 

Avoid a colour cacophony. (Multiple routers)
It is not desirable to give each profile its own colour.
With Brouter, the number of profile choices is quite large.
How many will be offered later is still unknown.

Coloured track sections are useful.
A colour change indicates so a customised profile section.
Highlighted sections are protected by a standard profile change.
Like the "beeline" section which should remain protected in an overall recalculation.
![image](https://github.com/devemux86/cruiser/assets/54167201/7d699511-cdc7-4618-9902-55204db7a419)
[Ardennen_beeline_src_guiding_x_dir.zip](https://github.com/devemux86/cruiser/files/11505636/Ardennen_beeline_src_guiding_x_dir.zip)
Result:
Default profile colour & custom profile colour & beeline colour.

**Comment by devemux86** [comment, marked as answer]
Different colors per route profile are available for BRouter, Kurviger in [Cruiser 3.0.20](https://play.google.com/store/apps/details?id=gr.talent.cruiser).

Available in "Settings | Routing | Appearance | Colors" or search settings for "colors".
- See the images in https://github.com/devemux86/cruiser/discussions/158#discussioncomment-5857870

**Reply by SF4User** [reply]
perfect !
